ci_framework/roles/os_must_gather/defaults/main.yml

```diff
@@ -20,4 +20,5 @@
 cifmw_os_must_gather_image: "quay.io/openstack-k8s-operators/openstack-must-gather:latest"
 cifmw_os_must_gather_output_dir: "{{ cifmw_basedir | default(ansible_user_dir ~ '/ci-framework-data') }}"
 cifmw_os_must_gather_timeout: "10m"
+cifmw_os_must_gather_additional_namespaces: "kuttl,sushy-emulator"
```

Comment:
Can we add additional namespaces here https://github.com/openstack-k8s-operators/ci-framework/blob/main/ci_framework/roles/os_must_gather/tasks/main.yml#L60 also under rescue?

Reply:
of course, doing that in my next iteration.

Reply:
Done